### PR TITLE
Finished the API

### PR DESCRIPTION
**Pull request by Weeezzt**
Filled the API with ten new, ten available and ten movies for kids

Screenshot 2023-12-22 at 17 27 15

### CODE REVIEW

**Review by Kookiie88 — APPROVED** [review]
Great work all the objects seem to have the correct properties and no values are missing.

**Comment by Kookiie88** [comment]
I would suggest too add the month that the movie is also released, 1 year a pretty big timespan.

File: movies.json [hunk 1]
```diff
@@ -1,16 +1,365 @@
...
+            "new": false,
+            "poster": "https://m.media-amazon.com/images/M/MV5BYWVkMWEyMDUtZTVmOC00MTYxLWE1ZTUtNjk4M2IzMjY2OTIxXkEyXkFqcGdeQXVyMDk5Mzc5MQ@@._V1_FMjpg_UX1000_.jpg"
+          },
+          {
+            "title": "Interstellar",
+            "director": "Christopher Nolan",
+            "release_year": 2014,
+            "genre": ["Äventyr", "Drama"],
+            "actors": ["Matthew McConaughey", "Anne Hathaway"],
+            "description": "",
```
Comment (AnnaLenaO) [inline movies.json:213]: Please add a description if necessary.

File: movies.json [hunk 2]
```diff
@@ -1,16 +1,365 @@
...
+            "new": false,
+            "poster": "https://m.media-amazon.com/images/M/MV5BMjExOTQ4MDMyMV5BMl5BanBnXkFtZTgwMTE3NDM2MzE@._V1_.jpg"
+          },
+          {
+            "title": "The Boss Baby",
+            "director": "Tom McGrath",
+            "release_year": 2017,
+            "genre": ["Animerad", "Äventyr", "komedi"],
+            "actors": ["Alec Baldwin", "Steve Buscemi"],
+            "description": "",
```
Comment (AnnaLenaO) [inline movies.json:321]: Please add a description if necessary.

File: movies.json [hunk 3]
```diff
@@ -1,16 +1,365 @@
...
+            "rating": 3,
+            "new": false,
+            "poster": "https://m.media-amazon.com/images/M/MV5BMTg3NzQ2NDgyNF5BMl5BanBnXkFtZTcwMDc1NzIyMw@@._V1_FMjpg_UX1000_.jpg"
+          },
+          {
+            "title": "The Polar Express",
+            "director": "Robert Zemeckis",
+            "release_year": 2004,
+            "genre": ["Animerad", "Äventyr"],
+            "actors": [""],
```
Comment (AnnaLenaO) [inline movies.json:356]: Please add actors if necessary.

**Review by AnnaLenaO — COMMENTED** [review]
Great work with all 10 properties. There are three missing values that can be added if necessary. 
According to the PBI it sholud be at least 10 upcoming, 10 child moovies & 10 available moovies. Now there are 5 moovies with release year 2024. If it is important for us with at least 10 upcoming , please change. Otherwise it can be like this :)

File: movies.json (hunk 3 above)
Comment (Weeezzt) [reply movies.json:356]: Fixed! :)

Comment (Weeezzt) [reply movies.json]: Fixed the missing info. There are 10 upcoming movies but 5 that has their world premiere after new year

File: movies.json (hunk 1 above)
Comment (Weeezzt) [reply movies.json]: Done

File: movies.json (hunk 2 above)
Comment (Weeezzt) [reply movies.json]: Done

**Comment by Weeezzt** [comment]
I've added a new key premiere to all movies with "new": true. It has a value of "Premiär 6 Januari" for example. I've also fixed some missing info that Anna-Lena found. There are 10 new/upcoming movies so i wont be changing or adding any more movies to the API. Please have another look at the changes and I'll listen if theres new feedback :)

**Review by EmilioSanchezLernia — APPROVED** [review]
Looks good, very informative API with plenty of opportunity to filter.

**Pull request merged**